### PR TITLE
September 2023 newsletter

### PR DESCRIPTION
**Pull request by davidfischer**

### CODE REVIEW

File: content/posts/newsletter-september-2023.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+## New features
+
+Our progress in September was:
+
+* We produced two content pieces about what happens
+  when a publisher on our network [truly goes viral]({filename}../posts/2023-going-truly-viral.md)
+  and one on [optimizing European ad campaigns in the GDPR era]({filename}../posts/2023-european-landing-pages-gdpr.md).
+* Our ad server hit v3.0 when we upgraded Django, a core dependency
+  added some pricing estimations for advertisers to get a better idea
+  of campaign prices when requesting a new campaign.
```
Comment (ericholscher) [inline content/posts/newsletter-september-2023.md:20]: This reads a bit oddly. Specifically "a core dependency added some pricing estimations for advertisers". Seems like maybe 2 different bullet points got merged together?

File: content/posts/newsletter-september-2023.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+* The advertiser reporting features that will help integrate our metrics with Tableau
+  to update performance metrics automatically, is continuing but very slowly.
+  We're aiming to collaborate with a few advertisers who use Tableau (or something similar)
+  to make sure we build exactly what our advertisers want.
+  If you're interested in this, please [get in touch]({filename}../pages/contact.md)!
+* We are making some improvements to better handle multiple ad placements on the same page.
+  Our policy is still to not have our ads beside competitors' ads
+  and to have just 1 ad at a time,
+  but there are some situations like very long pages where multiple ad placements make sense.
```
Comment (ericholscher) [inline content/posts/newsletter-september-2023.md:58]: Does this mean we're looking at updating our policy? The wording here is not super clear about what is actually happening.

**Review by ericholscher — APPROVED** [review]
👍

File: content/posts/newsletter-september-2023.md (hunk 1 above)
Comment (davidfischer) [reply content/posts/newsletter-september-2023.md:20]: Ya, definitely. I'll update it.

File: content/posts/newsletter-september-2023.md (hunk 2 above)
Comment (davidfischer) [reply content/posts/newsletter-september-2023.md:56]: I considered this a message to ease people into the change but happy to discuss more. Fundamentally, I don't think our position has changed (much). We don't want our ads to appear on pages full of ads. However, in some situations, having a 2nd ad can make sense.

Comment (ericholscher) [reply content/posts/newsletter-september-2023.md:56]: Maybe something like:

```suggestion
  Our policy is still to not have our ads beside competitors' ads
  and to have just 1 ad at a time,
  but there are some situations like very long pages where we allow multiple ad placements on a case by case basis.
```

**Pull request merged**